src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,91 @@
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background-color: var(--card-background-color);
+  border: 1px solid var(--card-border-color);
+  border-radius: 5px;
+}
+
+.product-card__pic {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.product-card__name {
+  margin-bottom: 4px;
+  line-height: 18px;
+
+  font-size: 12px;
+  color: var(--card-color-dark);
+  font-weight: 500;
+}
+
+.product-card__code {
+  margin-bottom: 16px;
+  display: block;
+
+  font-size: 10px;
+  color: var(--card-color-light);
+}
+
+.product-card__stars {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 16px;
```

Comment:
Try to rearrange it according to [this manual](https://codeguide.academy/html-css.html#css-order)
line-height is a text property